incubator-tools/child_entity_tag_using_header/child_entity_tag_using_header.ipynb

```diff
@@ -0,0 +1 @@
ument = documentai.Document.to_json(doc)\n","        utilities.store_document_as_json(document, output_bucket, filepath)\n","        print(\"\\tCompleted.\")\n","    print(\"Processing Completed for all files\\n\")\n","    if issue_files:\n","        print(f\"Below {count}-file(s) have some issues to process...\")\n","        print(issue_files)"]},{"cell_type":"markdown","id":"ba7feec5-06c7-4706-a992-50157f67ec78","metadata":{"id":"ba7feec5-06c7-4706-a992-50157f67ec78"},"source":["After Executed all above code cells, run below Entry-Function cell to start processing input files, Whose child entities need to tagged using header Keyword"]},{"cell_type":"code","execution_count":null,"id":"6d2238a2-b797-41ae-b7c4-e5665883d932","metadata":{"tags":[],"id":"6d2238a2-b797-41ae-b7c4-e5665883d932"},"outputs":[],"source":["child_entity_tag_using_header(\n","    gcs_input_path=GCS_INPUT_PATH,\n","    gcs_output_path=GCS_OUTPUT_PATH,\n","    list_total_amount=LIST_TOTAL_AMOUNT,\n","    total_amount_type=TOTAL_AMOUNT_TYPE,\n",")"]},{"cell_type":"markdown","id":"b02a0f30-9f93-46bf-b871-f34b4df08894","metadata":{"id":"b02a0f30-9f93-46bf-b871-f34b4df08894"},"source":["## 4. Output"]},{"cell_type":"markdown","id":"b4232146-0beb-4713-92ff-bf23606d45bf","metadata":{"id":"b4232146-0beb-4713-92ff-bf23606d45bf"},"source":["The items which are below the matched keyword will be tagged as entity name given"]},{"cell_type":"markdown","id":"5b486827-ddd2-488e-b9fb-5ac249d23e98","metadata":{"id":"5b486827-ddd2-488e-b9fb-5ac249d23e98"},"source":["<table>\n","    <tr>\n","        <td> Sample JSON-Input file, Entities Count(24)</td>\n","        <td> <img src=\"./images/before_child_entity_tag.png\" width=800 height=400> </td>\n","    </tr>\n","    <tr>\n","        <td> Sample JSON-Output file, Entities Count(25+4)</td>\n","        <td> <img src=\"./images/after_child_entity_tag.png\" width=800 height=400>  </td>\n","    </tr>\n","</table>"]},{"cell_type":"markdown","id":"d9a8c587-8d67-4953-b787-7e4c63a3a5ed","metadata":{"id":"d9a8c587-8d67-4953-b787-7e4c63a3a5ed"},"source":["JSON Sample output after tagging child-entities, **5-new entities** added to Document Proto\n","<img src=\"./images/after_child_entity_tag_json_results.png\" width=450 height=150></img>"]},{"cell_type":"code","execution_count":null,"id":"a9900525","metadata":{"id":"a9900525"},"outputs":[],"source":[]},{"cell_type":"code","source":[],"metadata":{"id":"l9AV-8GfCHs6"},"id":"l9AV-8GfCHs6","execution_count":null,"outputs":[]}],"metadata":{"environment":{"kernel":"python3","name":"common-cpu.m104","type":"gcloud","uri":"gcr.io/deeplearning-platform-release/base-cpu:m104"},"kernelspec":{"display_name":"Python 3","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.7.12"},"colab":{"provenance":[]}},"nbformat":4,"nbformat_minor":5}
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        return bucket, files_dir
</pre><p>You might be able to use the gcs_utilities function from document ai toolbox for this instead of reimplementing it. </p><p> </p><p><a href="https://cloud.google.com/python/docs/reference/documentai-toolbox/latest/google.cloud.documentai_toolbox.utilities.gcs_utilities#google_cloud_documentai_toolbox_utilities_gcs_utilities_split_gcs_uri" target="_blank">https://cloud.google.com/python/docs/reference/documentai-toolbox/latest/google.cloud.documentai_toolbox.utilities.gcs_utilities#google_cloud_documentai_toolbox_utilities_gcs_utilities_split_gcs_uri</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>